cli/bin/compile.js

```diff
@@ -12,7 +12,7 @@ module.exports = (file, options) => {
     if (options.graceful) {
       process.exit()
     } else {
-      process.exit(-1)
+      process.exit(1)
```

Comment:
Btw, my new ZSH setup (it reports the error code of the last command) helped me identify this issue because using `-1` actually caused a 255 error code instead of 1 like we were trying to do 😛

Reply:
Yeah—I think this one was my fault 😛